### PR TITLE
feat(FEC-12113): Add Side Panels Management to player UI

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

Expose public enums needed across plugins

**Related pr:**  
https://github.com/kaltura/playkit-js-ui-managers/pull/1
https://github.com/kaltura/kaltura-player-js/pull/544

solves: FEC-12113

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/reducers/shell.js [hunk 1]
```diff
@@ -55,6 +55,33 @@ export const SidePanelModes = {
   OVER: 'over'
 };
 
+export const ReservedPresetNames = {
+  Playback: 'Playback',
```
Comment (yairans) [inline src/reducers/shell.js:59]: There are more

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/reducers/shell.js [hunk 2]
```diff
@@ -55,6 +55,33 @@ export const SidePanelModes = {
...
+  SidePanelRight: 'SidePanelRight',
+  SidePanelBottom: 'SidePanelBottom',
+  PresetArea: 'PresetArea',
+  InteractiveArea: 'InteractiveArea',
+  PlayerArea: 'PlayerArea',
+  VideoArea: 'VideoArea'
+};
+
+export const RelativeToTypes = {
+  Before: 'Before',
```
[the remark this answers is not in the document]
Comment (JonathanTGold) [reply src/reducers/shell.js]: removed

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/reducers/shell.js (hunk 1 above)
Comment (yairans) [reply src/reducers/shell.js:59]: what about them?

Comment (JonathanTGold) [reply src/reducers/shell.js:59]: fixed

**Review by yairans — APPROVED** [review]

**Pull request merged**